src/CodeStyle/Core/Analyzers/Microsoft.CodeAnalysis.CodeStyle.csproj

```diff
@@ -24,8 +30,152 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.CodeStyle.UnitTests" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\..\..\Compilers\Core\Portable\InternalUtilities\EnumerableExtensions.cs" Link="Formatting\EnumerableExtensions.cs" />
```

Comment:
Not required for this PR, but you probably want to consider creating a shared project at some point to keep the project files sane.

Reply:
If things keep growing it will be more viable, but I'm hoping some of this functionality starts to be consolidated over time.